### PR TITLE
Document enabling an Nvidia GPU in K3S

### PR DESCRIPTION
**Pull request by pbronneberg**
As discussed with @rdoxenham

### CODE REVIEW

File: docs/quickstart/nvidia-slemicro.md [hunk 1]
```diff
@@ -315,7 +315,107 @@ exit
...
+### Restart K3S
+
+Ensure K3S is restarted and discovers the NVidia Runtime.
+
+> Rationale: During boot K3S discovers the Nvidia runtime, and register this as an available runtime.
+
+Verify the runtime is registered:
+
+```
+cat /var/lib/rancher/k3s/agent/etc/containerd/config.toml | grep nvidia
```
Comment (e-minguez) [inline docs/quickstart/nvidia-slemicro.md:331]:
```suggestion
grep nvidia /var/lib/rancher/k3s/agent/etc/containerd/config.toml
```

File: docs/quickstart/nvidia-slemicro.md [hunk 2]
```diff
@@ -315,7 +315,107 @@ exit
...
+```
+
+This should show multiple lines referencing the newly installed runtime.
```
Comment (e-minguez) [inline docs/quickstart/nvidia-slemicro.md:334]: Could you add an example?

**Comment by pbronneberg** [comment]
Sorry.. missed the changes. Thanks for the reminder. Will look for an example. Agree with the proposals

**Comment by agracey** [comment]
Closing due to alternative merged (please let me know if there's something new that we need to pull in as well!)

**Pull request closed without merging**